parity/export_hardcoded_sync.rs

```diff
@@ -0,0 +1,123 @@
+// Copyright 2015-2019 Parity Technologies (UK) Ltd.
```

Comment:
Is this related to Prometheus?

Reply:
Dead code left after merge. Will be removed.